lib/HunkView.ml

```diff
@@ -31,44 +110,54 @@ let ui_hunk_summary (hunk : Patch.hunk) : Nottui.ui =
       at_symbols;
     ]
 
-let rec process_lines (mine_num : int) (their_num : int) (acc : Nottui.ui list)
-    = function
+let process_line (hunk : Patch.hunk) i line =
+  let line_number =
+    W.string
+      ~attr:Notty.A.(fg yellow)
+      (Printf.sprintf "%4d " (succ (hunk.Patch.mine_start + i)))
```

Comment:
This removes the "their" line number. I think that's not right.

Reply:
This one is a big one:

Alan: Removed "their" number lines because the word-diff is now in one line
Jules: We want to keep the + and - lines and word-diff into these + and - lines the word highlighting 

Word-diff is something on top of regular diff view -> not something different